test/perf/perf_simple_query.cc

```diff
@@ -427,6 +427,12 @@ static std::vector<perf_result> do_cql_test(cql_test_env& env, test_config& cfg)
                 .build();
     }).get();
 
+    std::cout << "Disabling auto compaction" << std::endl;
+    env.db().invoke_on_all([] (auto& db) {
+        auto& cf = db.find_column_family("ks", "cf");
+        return cf.disable_auto_compaction();
+    }).get();
+
```

Comment:
Maybe we should wait for compaction to complete?

Reply:
Doe it _start_ at all before test completes population and starts testing? :) I don't think anybody ever faced that problem without tablets _and_ without the newly introduced --memtable-partitions option, so it's likely run with all data in memtable _or_ with a single sstable on disk. In general I agree that we might want to wait for compaction to complete, but I think that compaction needs better control than just "wait completion"

Reply:
disable_auto_compaction stops ongoing compactions and waits for them since c138981286cef07e61fb41c075bc25f15cb8c9c8